src/main/groovy/life/qbic/datamodel/dtos/business/AffiliationCategory.groovy

```diff
@@ -0,0 +1,48 @@
+package life.qbic.datamodel.dtos.business
+
+/**
+ * We define three major business affiliation categories here, which
+ * means that we decide between
+ *  - internal: An affiliation we consider as within the University of Tübingen or University
+ *  Hospital of Tübingen
+ *  - external academic: An outside affiliation but an academic institution (public research
+ *  institutions)
+ *  - external: An outside affiliation but not academic (i.e. private sector, companies, etc)
+ */
+enum AffiliationCategory {
+
+  INTERNAL("internal"),
+  EXTERNAL_ACADEMIC("external academic"),
+  EXTERNAL("external non-academic")
```

Comment:
In the diagram we specified this enum value as Industry, maybe a combination solution would be the best to drive the point home? 
```suggestion
  EXTERNAL_INDUSTRY("external industrial")
```

Reply:
Sorry, I forgot to update the ER diagram after talking to @wow-such-code .

Reply:
Ah Ok! That explains it 👍